packages/docs-reanimated/docs/scroll/useAnimatedScrollHandler.mdx

```diff
@@ -4,126 +4,119 @@ sidebar_position: 3
 
 # useAnimatedScrollHandler
 
-import DocsCompatibilityInfo from '../_shared/_docs_compatibility_info.mdx';
+`useAnimatedScrollHandler` is a hook that returns an event handler reference. It can be used with React Native's scrollable components.
 
-<DocsCompatibilityInfo />
+## Reference
 
-This is a convenience hook that returns an event handler reference which can be used with React Native's scrollable components.
+```js
+import { useAnimatedScrollHandler } from 'react-native-reanimated';
+
+function App() {
+  const offsetY = useSharedValue(0);
+
+  // highlight-start
+  const scrollHandler = useAnimatedScrollHandler((event) => {
+    translationY.value = event.contentOffset.y;
```

Comment:
`offsetY` or `translationY`? Choose one, not both 😄

Reply:
big shame, changing now